### DISCUSSION TITLE
Why are the layer heights for the 0.6 mm nozzle so low?

### DISCUSSION THREAD

**Discussion by jjramsey** [phnxdrgn536/FLSun-v400#28] [General]
If I look at the layer heights for the 0.4 mm nozzle, there's a wide set of possibilities, from 0.06 mm to 0.3 mm. However, for the 0.6 mm nozzle, the layer height maxes out at 0.15 mm, and the "Normal" layer height is the same as that of the 0.4 mm nozzle -- 0.15 mm -- which negates much of the advantage of using a 0.6 mm nozzle in the first place.

**Comment by phnxdrgn536** [comment]
All profiles are constantly being fine tuned and adjust to achieve the best
print qualities for the intended use, sadly since most every printer
default nozzle size is 0.4mm the 0.6 and 0.8 nozzle profiles do not get as
much attention when it comes to adjustments. If you have any suggestions
for the 0.6 nozzle profiles please let me know and I will happily try and
add those adjustments.

**Reply by jjramsey** [reply]
When I last made a profile for the 0.6 mm nozzle, before I even knew of the profiles here, I just followed the guidance here from Matthew Herman: https://m.facebook.com/photo.php?fbid=5733350736785053&id=100003305507114&set=p.5733350736785053&mibextid=Nif5oz

Unfortunately, the photo cuts off the names of the fields. I don't know what he set the wall thickness to. You may want to ask him for more details.

**Comment by jjramsey** [comment]
If I look at PrusaSlicer's settings for a Prusa with a 0.6 mm nozzle, it has two "DETAIL" settings, one with a 0.15 mm layer height and one with a 0.2 mm layer height, a "NORMAL" setting with a 0.3 mm layer height, a "SPEED" setting with a 0.35 mm layer height, and a "DRAFT" setting with a 0.4 mm layer height. I think that gives some reasonable guidance for what would be sensible layer heights for that nozzle diameter. That would also track with Matthew Herman's Cura profile.

**Comment by phnxdrgn536** [comment]
Awesome, I’ll get on tuning those
